lib/shared/src/chat/transcript/index.ts

```diff
@@ -129,6 +129,13 @@ export class Transcript {
         })
     }
 
+    public removeDisplayTextFromLastInteraction(): void {
+        if (!this.getLastInteraction()) {
+            return
+        }
+        this.getLastInteraction()?.removeDisplayText()
```

Comment:
The way `?.` works, the call won't happen if `getLastInteraction()` returns undefined. So you can simplify this. (Personally, I find this aspect of TypeScript hard to read, because the meaning of:

```
this.foo()?.bar().baz()
```

and

```
const x = this.foo()?.bar()
x.baz()
```

are different, and JavaScript does not work this way. But oh well.)

```suggestion
        this.getLastInteraction()?.removeDisplayText()
```